src/components/SearchForm/SearchFormOptions.tsx

```diff
@@ -0,0 +1,132 @@
+import MULTIPLE_MYELOMA_CANCERTYPE_CODES from '../../queries/mockData/MultipleMyelomaCancerTypeCodes.json';
+import MULTIPLE_MYELOMA_MEDICATION_CODES from '../../queries/mockData/MultipleMyelomaMedicationCodes.json';
+import PROSTATE_CANCER_TYPE_CODES from '../../queries/mockData/prostateCancerTypeCodes.json';
+
+export type CancerTypeDetails = {
+  category: string;
+  cancerCodes: CodedValueType[];
+  cancerSubtype: CodedValueType[];
+  biomarkers: CodedValueType[];
+  surgeryCodes: CodedValueType[];
+  medications: CodedValueType[];
+  stages: CodedValueType[];
+  radiationCodes: CodedValueType[];
+};
+
+export const cancerTypeDetails: Record<string, CancerTypeDetails> = {
+  lung: buildCancerCodeJSON(
+    'lung',
+    LUNG_CANCER_TYPE_CODES.entry,
+    LUNG_CANCER_SUBTYPE_CODES.entry,
+    LUNG_MEDICATION_CODES.entry,
+    LUNG_RADIATION_CODES.entry,
+    LUNG_SURGERY_CODES.entry,
+    BREAST_CANCER_STAGES.entry,
+    BREAST_CANCER_BIOMARKERS.entry
+  ),
+  colon: buildCancerCodeJSON(
+    'colon',
+    COLON_CANCER_TYPE_CODES.entry,
+    null,
+    COLON_MEDICATION_CODES.entry,
+    COLON_RADIATION_CODES.entry,
+    COLON_SURGERY_CODES.entry,
+    BREAST_CANCER_STAGES.entry,
+    null
+  ),
+  brain: buildCancerCodeJSON(
+    'brain',
+    BRAIN_CANCER_TYPE_CODES.entry,
+    BRAIN_CANCER_SUBTYPE_CODES.entry,
+    BRAIN_MEDICATION_CODES.entry,
+    BRAIN_RADIATION_CODES.entry,
+    BRAIN_SURGERY_CODES.entry,
+    BREAST_CANCER_STAGES.entry,
+    BRAIN_BIOMARKER_CODES.entry
+  ),
+  prostate: buildCancerCodeJSON(
+    'prostate',
+    PROSTATE_CANCER_TYPE_CODES.entry,
+    null,
+    null,
+    null,
+    null,
+    BREAST_CANCER_STAGES.entry,
+    null
+  ),
+  breast: buildCancerCodeJSON(
+    'breast',
+    BREAST_CANCER_TYPE_CODES.entry,
+    null,
+    BREAST_CANCER_MEDICATIONS.entry,
+    null,
+    null,
+    BREAST_CANCER_STAGES.entry,
+    BREAST_CANCER_BIOMARKERS.entry
+  ),
+  mm: buildCancerCodeJSON(
+    'mm',
+    MULTIPLE_MYELOMA_CANCERTYPE_CODES.entry,
+    null,
+    MULTIPLE_MYELOMA_MEDICATION_CODES.entry,
+    null,
+    null,
+    BREAST_CANCER_STAGES.entry,
+    null
+  ),
+};
+
+export const cancerTypeOptions: CodedValueType[] = Object.values(cancerTypeDetails).reduce<CodedValueType[]>(
+  (codes: CodedValueType[], details) => {
+    codes.push(...details.cancerCodes);
+    return codes;
+  },
+  []
+);
+
+function buildCancerCodeJSON(
+  category: string,
+  cancerCodes: CodedValueType[],
+  subTypeCodes: CodedValueType[],
+  medicationCodes: CodedValueType[],
+  radiationCodes: CodedValueType[],
+  surgeryCodes: CodedValueType[],
+  stages: CodedValueType[],
+  biomarkerCodes: CodedValueType[]
+) {
+  const details: CancerTypeDetails = {
+    category: category,
+    cancerCodes: cancerCodes,
+    cancerSubtype: subTypeCodes,
+    biomarkers: biomarkerCodes,
+    medications: medicationCodes,
+    radiationCodes: radiationCodes,
+    surgeryCodes: surgeryCodes,
+    stages: stages,
+  };
+
+  return details;
+}
```

Comment:
I don't understand the point of this function. Why not just use the object it declares directly?

Reply:
removed function and created the objects directly